Bonsai.NuGet.Design/PackageSourceConfigurationDialog.cs

```diff
@@ -23,6 +26,15 @@ public PackageSourceConfigurationDialog(IPackageSourceProvider sourceProvider)
             provider = sourceProvider;
         }
 
+        protected override void ScaleControl(SizeF factor, BoundsSpecified specified)
```

Comment:
Is there a reason I'm missing that this is in `ScaleControl`? It seems like it should just be at the end of the constructor since you're effectively imitating the old `ImageList` deserialization.

Even if `ScaleControl` is the appropriate place for this, the images will be added to the list every single time it is called which seems incorrect.

Reply:
This is to ringfence in case we ever need to scale images before adding them to the list, the code will already be in the right place, the same way it is done for `StartScreen`, but you are right about correcting for multiple calls, I will fix in all places where this pattern is used.

Reply:
I ended up following the pattern in the other form classes where this scaling is done, which is simply to clear the list before adding the images. We can discuss and overhaul this pattern as a whole if needed later when we get to tweaking the scale factors for .NET 8.